web-app/src/api/translator.js

```diff
@@ -0,0 +1,25 @@
+import api from ".";
+
+export const formatReport  = async({messages, full}) => {
+    const { timestamp, author, content } = messages[0].value;
+    const { node, common_issue, title, body}  = content
+    const user = await api.account.get(author);
+    return {
+        id: messages[0].key,
+        timestamp,
+        user,
+        status: 'open',
```

Comment:
Did you forget this one or there is something in progress relating the report's status?

Reply:
wip for now